cookbooks/aws-parallelcluster-install/resources/efs/efs_redhat8.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+provides :efs, platform: 'redhat' do |node|
+  node['platform_version'].to_i == 8
+end
+unified_mode true
+
+use 'partial/_install_efs_utils_rhel' unless redhat_ubi?
+
+default_action :install_utils
+
+action :install_utils do
+  # If on RHEL8 Docker, we don't install the base packages, so install_utils cannot be built.
+end if redhat_ubi?
```

Comment:
Nitpick: could this logic be moved to `partial/_install_efs_utils_rhel` in order to improve readability?

Reply:
Done